src/effects/VST3/VST3Instance.cpp

```diff
@@ -153,7 +154,7 @@ bool VST3Instance::ProcessFinalize() noexcept
 
 bool VST3Instance::ProcessInitialize(EffectSettings &settings, double sampleRate, ChannelNames chanMap)
 {
-   if(mWrapper->Initialize(settings, sampleRate, Steinberg::Vst::kRealtime, mProcessingBlockSize))
+   if(mWrapper->Initialize(settings, sampleRate, Steinberg::Vst::kOffline, mProcessingBlockSize))
```

Comment:
I understand the second of the six commits, except for this unexplained change.  What does it accomplish?

Reply:
Some VST3 plugins can offer better processig quality in `kOffline` mode. I think it used to be `kOffline` some time before refactoring, but I've made a typo

Reply:
So then, will you revert that change or comment that you are keeping it?

Reply:
No, I think `kOffline` fits better for destructive processing. `RealtimeInitialize` uses `kRealtime` mode.